ipython/Enwiki Revert Classifier.ipynb

```diff
@@ -0,0 +1,301 @@
+       "text": [
+        "<matplotlib.figure.Figure at 0x7f5447ba4f60>"
+       ]
+      }
+     ],
+     "prompt_number": 6
+    },
+    {
+     "cell_type": "markdown",
+     "metadata": {},
+     "source": [
+      "And there you have it. "
+     ]
+    },
+    {
```

Comment:
It seems like this is something that ipython notebook does when you ask it execute the last block.  If you feel strongly, I'll remove it, but it seems to me that we should let this kind of issue pass.